.gitignore

```diff
@@ -10,6 +10,3 @@ views/partials/changelog.hbs
 .vscode
 config
 coverage
-public/github-markdown.css
-public-back
-/public
```

Comment:
Is it on purpose that /public is being added to Git? It seems like a slightly strange workflow to check that stuff in, but I could see it being useful if you want to use the packaged version...

Reply:
This is intentional, `favicon.ico`, `robots.txt`, are not generated, and are served by the express server in the deployed React app.

I think the generated `.js` and `.css` could be removed.